react/features/polls/components/web/PollCreate.js

```diff
@@ -73,6 +73,14 @@ const PollCreate = (props: AbstractProps) => {
     });
 
     const onQuestionKeyDown = useCallback(ev => {
+        // keyCode 229 means that user pressed some button,
```

Comment:
This looks like it would fit inside checkModifiers. Maybe we can rename it to `ignoreEvent` ?

Reply:
I tried your suggestion, added the check inside an ignoreEvent(ev) function, called it inside checkModifiers, after that I called it inside onQuestionKeyDown and onAnswerKeyDown, but it doesn't work.

Reply:
did you return true if the condition was met?

Reply:
Yes. It seems that the only solution way was this one. Updated the PR.